web/src/filter.ts

```diff
@@ -862,6 +869,10 @@ export class Filter {
     generate_redirect_url(): string {
         const term_types = this.sorted_term_types();
 
+        if (_.isEqual(term_types, ["sender", "search", "has-reaction"])) {
+            return "/#narrow/has/reaction/sender/me";
+        }
```

Comment:
Can you explain this logic? It seems like it ignores the actual operand for the current view, so if we were looking at reactions sent by some other user, this would switch it to us?

Reply:
Yes, I have changed it to also check `filter.operands("sender")[0] === people.my_current_email()` wherever required.